tests/contracts/base/test_issuer.py

```diff
@@ -86,9 +86,16 @@ def calculate_second_phase_reward(locked, total_locked, locked_periods):
 
     # Only owner can initialize
     with pytest.raises((TransactionFailed, ValueError)):
-        tx = issuer.functions.initialize(0).transact({'from': staker})
+        tx = issuer.functions.initialize(0, staker).transact({'from': staker})
         testerchain.wait_for_receipt(tx)
-    tx = issuer.functions.initialize(economics.erc20_reward_supply).transact({'from': creator})
+
+    # Only owner can initialize
+    with pytest.raises((TransactionFailed, ValueError)):
+        tx = issuer.functions.initialize(economics.erc20_reward_supply, staker).transact({'from': creator})
+        testerchain.wait_for_receipt(tx)
```

Comment:
I guess here should be other comment, something related to that staker has no tokens